services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/FlowPingSpec.groovy

```diff
@@ -224,7 +224,30 @@ class FlowPingSpec extends BaseSpecification {
     }
 
     def getOfSwitchCombinations() {
```

Comment:
If possible, please change method name to getOFSwitchCombinations(). Also, would be great to move code comments on the switches quirks to the groovydoc.

Reply:
`getOFSwitchCombinations` is incorrect camel-case name. Won't do.
Will add a brief groovydoc describing what this method does.

Reply:
please see this guide (we are following Google style guide on Kilda)
http://google.github.io/styleguide/javaguide.html#s5.3-camel-case
p.3